manimlib/mobject/svg/brace.py

```diff
@@ -9,7 +9,7 @@
 from manimlib.mobject.types.vectorized_mobject import VMobject
 from manimlib.utils.config_ops import digest_config
 from manimlib.utils.space_ops import get_norm
-
+import copy
```

Comment:
this is better suited in a general bug fix PR (so we have a proper organization)

Reply:
Again, this really does not belong here. Can I easily remove this commit from my PR?

Reply:
Not *easily*, unless you are very comfortable with git.

Reply:
I am somehow comfortable with git, but mainly use it for own projects where I am the sole author. So I might need some guidance when it comes to stuff like this. (I actually have no idea why these commits show up in my PR, because I just pulled from this master on Github and then created a new branch.)

Reply:
So I think what you need to do is identify the rogue commit that added these changes without you knowing, and then rebase your branch.

An easier way to go would be to close this PR, and delete your changes from your local repo (maybe `git stash` them?). Then make sure your repo is at the same state as the current master branch (by pulling, or by nuking your repo and cloning again if necessary). Then add your changes back. Then PR again, making sure that this spurious changes are not there.

Reply:
@PhilippImhof I added the suggestion so you can easily remove it